src/pages/workspaces/workspace/analysis/AnalysisLauncher.js

```diff
@@ -249,6 +249,8 @@ const PreviewHeader = ({
   queryParams, runtime, readOnlyAccess, onCreateRuntime, analysisName, currentFileToolLabel, workspace, setCreateOpen, refreshRuntimes,
   workspace: { canShare, workspace: { cloudPlatform, namespace, name, bucketName, googleProject, workspaceId } }
 }) => {
+  const { ExportAnalysisModal } = analysisComponentDependencies.get()
```

Comment:
minimal edge adaption for rest of codebase that is not ready to fully participate in Composable DI framework.